### PR TITLE
Added new options: fixed-shards and archival-nodes

### PR DESCRIPTION
**Pull request by mm-near**
Added support for two new options in localnet creation:
- setting archival node
- fixed_shards - that would create genesis file with accounts that belong to each shard.

see nearcore/6658 PR that adds those options.

### CODE REVIEW

**Review by marcelo-gonzalez — APPROVED** [review]

File: tox.ini [hunk 1]
```diff
@@ -44,7 +44,8 @@ disable=missing-module-docstring,
         duplicate-code,
         too-many-arguments,
         consider-using-with,
-        unspecified-encoding
+        unspecified-encoding,
+	      too-many-locals,
```
Comment (mina86) [inline tox.ini:48]: Alternatively, add `# pylint: disable=too-many-locals` in function where this is the issue.

File: nearuplib/nodelib.py [hunk 2]
```diff
@@ -207,6 +214,10 @@ def run_binary(path,
         command.extend(['--n', str(non_validators)])
     if boot_nodes:
         command.extend(['--boot-nodes', boot_nodes])
+    if fixed_shards:
+        command.extend(['--fixed-shards'])
+    if archival_nodes:
+        command.extend(['--archival-nodes'])
```
Comment (mina86) [inline nearuplib/nodelib.py]: Those should just be `command.append('--fixed-shards')` etc.

**Review by mina86 — APPROVED** [review]

File: nearuplib/nodelib.py (hunk 2 above)
Comment (mm-near) [reply nearuplib/nodelib.py]: done.

File: tox.ini (hunk 1 above)
Comment (mm-near) [reply tox.ini:48]: ack

**Pull request merged**